Assignment/ComplexTest.c

```diff
@@ -0,0 +1,85 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Vinitha Perumal - GET
+// ------------------------------------------------------------------
+// ComplexTest.c
+// Program on Test1.2 branch.
+// ------------------------------------------------------------------------------------------------
+#include "Complex.h"
+#include <stdio.h>
+#include <math.h>
+
+void DisplayRes (ComplexNumber num);
+void TestCases ();
+
+void DisplayRes (ComplexNumber num) {
+   num.img < 0 ? printf ("%.2f - %.2fi\n", num.real, -num.img)
+      : printf ("%.2f + %.2fi\n", num.real, num.img);

```

Comment:
```suggestion
   printf ("%.2f %s %.2fi\n", a.real, a.img < 0 ? "-" : "+", a.img);

```
If there is a part of code that is being shared across snippets, try to take it out